libc/src/math/generic/powf.cpp

```diff
@@ -528,7 +528,7 @@ LLVM_LIBC_FUNCTION(float, powf, (float x, float y)) {
   // So if |y| > 151 * 2^24, and x is finite:
   //   |y * log2(x)| = 0 or > 151.
```

Comment:
I am not sure whether this comment needs to be updated

Reply:
Actually there are still more exact cases to be tested, those that are related to `y = 2^-n`, i.e. taking `2^n`-root of `x`.  At least we still need to clear the `FE_INEXACT` flag for those cases.  I'll update the comment when we have a complete list of all the exact cases for `powf`.